### PR TITLE
Week3 val

### PR DESCRIPTION
**Pull request by ValC001**
# Submitting Week 3 Assessment

### ✅ Completed Files

Mark the following as confirmation that the files in this repository are complete.

- [x] Coding Challenges
- [x] Interview Questions
- [x] React Debugging

### 📝 Feedback

If you would like specific feedback on a particular problem please leave details here:

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -10,9 +10,25 @@
...
+  it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.",()=>{
+    const fibLength1 = 6
+    const fibLength2 = 10
+     expect(fibonacciReturner(fibLength1).toEqual(1,1,2,3,5,8))
+     expect(fibonacciReturner(fibLength2).toEqual(1,1,2,3,5,8,13,21,34,55))
+  })
+})
 
 // a) Create a test with expect statements for each of the variables provided.
+// Pseudo Code
```
Comment (Genemichael25) [inline code-challenges.test.js]: Missing pseudocode and function attempt for number 1.

File: code-challenges.test.js [hunk 2]
```diff
@@ -47,21 +145,82 @@ const studyMinutesWeek2 = {
...
 
 const accountTransactions2 = [250, -89, 100, -96]
-// Expected output: [250, 161, 261, 165]
+// // Expected output: [250, 161, 261, 165]
 
 const accountTransactions3 = []
-// Expected output: []
+// // Expected output: []
+
+// // b) Create the function that makes the test pass.
```
Comment (Genemichael25) [inline code-challenges.test.js:245]: Same as # 1, pseudocode and function attempt.

**Review by Genemichael25 — CHANGES_REQUESTED** [review]
Valeria, I am requesting changes for the missing pseudocode and attempts on code challenges # 1 and 3. Please let me know when you have made the changes so that I can finish reviewing your assessment. Thanks

File: code-challenges.test.js [hunk 3]
```diff
@@ -66,33 +72,33 @@ const fibLength2 = 10
...
+// }
     
-     expect(studyMinutesArranger(studyMinutesWeek1).toEqual(15,15,20,30,30,60,90))
-     expect(studyMinutesArranger(studyMinutesWeek2).toEqual(10,15,20,45,60,65,100))
-  })
-})
+//      expect(studyMinutesArranger(studyMinutesWeek1).toEqual(15,15,20,30,30,60,90))
+//      expect(studyMinutesArranger(studyMinutesWeek2).toEqual(10,15,20,45,60,65,100))
+//   })
+// })
```
Comment (SunkissedQueen) [inline code-challenges.test.js]: The test and the function should not be commented out. They need to be visible for the test suite to see both.

File: code-challenges.test.js [hunk 4]
```diff
@@ -18,17 +18,23 @@
...
-// Pseudo Code
+// describe("fibonacciReturner",()=>{
+//   it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.",()=>{
+//     const fibLength1 = 6
+//     const fibLength2 = 10
+//      expect(fibonacciReturner(fibLength1).toEqual(1,1,2,3,5,8))
+//      expect(fibonacciReturner(fibLength2).toEqual(1,1,2,3,5,8,13,21,34,55))
+//   })
+// })
+
```
Comment (SunkissedQueen) [inline code-challenges.test.js]: should not be commented out

File: code-challenges.test.js [hunk 5]
```diff
@@ -22,10 +44,93 @@ const fibLength2 = 10
 
 // b) Create the function that makes the test pass.
 
-// --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
+
```
Comment (SunkissedQueen) [inline code-challenges.test.js]: missing the function

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Missing function on 1b

**Review by SunkissedQueen — APPROVED** [review]
Valeria, thanks for providing the changes. I appreciate that you are working through your novice skillsets. I am glad that you have accepted mentorship. Suggestions on things to work on: Pseudocode, Functions, and Content for interview questions.
Your tests have good structure. It looks like you need more content in your pseudocode about the data so you can create your function properly. Let's try to meet up and discuss more strategies that can support you as you are learning to develop.
Next assessment: provide more content on the pseudocode and your interview answers. With practice and knowledge of your blockers, you can be at advanced beginner by internship.

**Comment by ValC001** [comment]
Done

**Pull request merged**